.github/workflows/inductor_xpu_ut_nightly.yml

```diff
@@ -0,0 +1,203 @@
+name: inductor-xpu-ut-test
+
+on:
+  workflow_call:
+    inputs:
+      torch_xpu_ops_update:
+        required: false
+        type: string
+        default: 'true'
+        description: True means update xpu_ops when building pytorch, otherwise means not
+      expected_fail_num:
```

Comment:
I think we don't need this parameter for UT test. Please add a parameter for UT test scope, default value include example, extend UT, xpu op UT

Reply:
Done